### PR TITLE
[Review Needed]Pytest Style core/test_updating_atomgroup.py

### PR DESCRIPTION
**Pull request by utkbansal**
Fixes #

Changes made in this Pull Request:
 - 

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/core/test_updating_atomgroup.py [hunk 1]
```diff
@@ -40,84 +34,105 @@
...
+
+    @pytest.fixture()
+    def ag_updating(self, u):
+        return u.select_atoms("prop x < 5 and prop y < 5 and prop z < 5",
+                              updating=True)
+
+    @pytest.fixture()
+    def ag_updating_compounded(self, u, ag):
+        return u.select_atoms("around 2 group sele",
+                                   sele=ag, updating=True)
```
Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_updating_atomgroup.py]: indentation

File: testsuite/MDAnalysisTests/core/test_updating_atomgroup.py [hunk 2]
```diff
@@ -40,84 +34,105 @@
...
-        ag_static2 = self.ag_updating.select_atoms("all") 
-        assert_array_equal(ag_static1.indices, self.ag.indices)
-        assert_array_equal(ag_static2.indices, self.ag.indices)
-        self.u.trajectory.next()
-        assert_array_equal(ag_static1.indices, self.ag.indices)
-        assert_array_equal(ag_static2.indices, self.ag.indices)
-
-    def test_kwarg_check(self):
+        assert_equal(ag_updating_chained.indices,
+                           ag_updating_compounded.indices)
```
Comment (kain88-de) [inline testsuite/MDAnalysisTests/core/test_updating_atomgroup.py]: indentaion

**Comment by kain88-de** [comment]
Please fix all indentation issues. There are more to be found in the diff. I haven't marked them all.

**Comment by kain88-de** [comment]
@utkbansal please fix the indentations

**Comment by utkbansal** [comment]
@kain88-de Is this good to go or is there something else to be done?

**Pull request merged**